source/jormungandr/jormungandr/default_settings.py

```diff
@@ -158,6 +158,9 @@
 
 GREENLET_POOL_SIZE = int(os.getenv('JORMUNGANDR_GEVENT_POOL_SIZE', 10))
 
+# Number of greenlets simultaneously used for Real-Time proxies calls
+GREENLET_RT_POOL_SIZE = int(os.getenv('JORMUNGANDR_GEVENT_RT_POOL_SIZE', 10))
```

Comment:
It also need to be configurable by rt proxy instances

Reply:
As discussed, the number of greenlets is now configurable for each instance in the db